tpl-implementations/kokkos-kernels/include/exec_policy_wrapper_kk.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef EXEC_POLICY_WRAPPER_HPP_
```

Comment:
Would you consider using a bit more namespaced of a use-once macro name here?

Reply:
yeah for sure ...

Reply:
done